src/MSAL.PCL/Internal/Http/HttpClientFactory.cs

```diff
@@ -35,7 +35,17 @@ internal class HttpClientFactory
         private static HttpClient _client;
         private static readonly object LockObj = new object();
         public static bool ReturnHttpClientForMocks { set; get; }
+        public const long MaxResponseContentBufferSizeInBytes = 1024*1024;
 
+        private static HttpClient CreateHttpClient()
```

Comment:
creating HttpClient instances could result in out of sockets exception. guidance says that create a single instance and use it for the lifetime of the application.